index.js

```diff
@@ -0,0 +1,30 @@
+const taskInput = document.getElementById('myInput');
+const taskList = document.getElementById('taskList');
```

Comment:
DOM을 담고있는 변수는 $taskInput 혹은 taskInputElem과 같이 일반적인 변수와 차별성을 두면 좋아요.
추가적으로 변수명과 html의 id명을 맞춰주면 더 좋을 것 같아요!

```suggestion
const $taskInput = document.getElementById('taskInput');
const $taskList = document.getElementById('taskList');
```

Reply:
[93581c6](https://github.com/kzh4295/js-todo-app/pull/2/commits/93581c6585c7e5b4a868a9061558c457e56cedf0)에서 수정했습니다!

Reply:
아직은 커밋이 안올라와 있는데 커밋이 올라오면 알맞은 커밋 해시로 수정해주세요~

Reply:
넵 수정했습니닷!